### DISCUSSION TITLE
希望taro能够集成流式输出

### DISCUSSION THREAD

**Discussion by dragonforce2010** [NervJS/taro#15456] [Feature Request]
我的场景是，我在dify构建了一个机器人，然后我想在taro中调用dify的流式api和机器人对话，但目前taro不支持流式响应，很痛苦。各位大神有绕过且轻量级的方案吗？

**Comment by dragonforce2010** [comment]
dify的流式api请求是这样的
curl -X POST 'http://localhost/v1/chat-messages' \
--header 'Authorization: Bearer app-r3oA8T4xmBJbL2FWfcG4rWkA' \
--header 'Content-Type: application/json' \
--data-raw '{
    "inputs": {},
    "query": "What are the specs of the iPhone 13 Pro Max?",


    "response_mode": "streaming",
    "conversation_id": "",
    "user": "abc-123"
}'

**Reply by liuyisnake** [reply]
估计搞不成，微信小程序就没有XMLHttpRequest，微信自己的wx.request也不支持实时读取stream

**Comment by lou0820** [comment]
我们也是使用dify+taro做ai客服的页面，我是调用的原生wx.request的方法 onChunkReceived   可以实现流式响应。不清楚你说的是不是这个意思
